tasks/__init__.py

```diff
@@ -82,6 +82,9 @@
 from tasks.unit_tests import invoke_unit_tests
 from tasks.update_go import go_version, update_go
 from tasks.windows_resources import build_messagetable
+from .custom_task import custom__call__
```

Comment:
```suggestion
from tasks.custom_task.custom_task import custom__call__
```
We should prefer full paths to avoid surprises

Reply:
Good point

Reply:
Fixed with [43c0deb](https://github.com/DataDog/datadog-agent/pull/25147/commits/43c0deb36a8685612785cbac2f354eb8737b4c95) !